csm/conf/setup.py

```diff
@@ -160,8 +185,13 @@ def _config_user(self, reset=False):
         """
         if not reset:
             if not self._is_user_exist():
-                Setup._run_cmd("useradd -d "+const.CSM_USER_HOME+" -p "+self._password+" "+ self._user)
+                _password = self._fetch_csm_user_password(decrypt=True)
```

Comment:
Will these code changes work in DEV environment?

Reply:
No these are not expected.